cmd/podman/generate/systemd.go

```diff
@@ -78,6 +78,10 @@ func init() {
 	flags.BoolVar(&systemdOptions.New, newFlagName, false, "Create a new container or pod instead of starting an existing one")
 	flags.BoolVarP(&systemdOptions.NoHeader, "no-header", "", false, "Skip header generation")
 
+	extraHeaderFlagName := "extra-header"
+	flags.StringVar(&systemdOptions.ExtraHeader, extraHeaderFlagName, "", "Append extra header")
```

Comment:
I would be fine with making this and `--no-header` conflict - so if `--no-header` is set to true and `--extra-header` is set, error

Reply:
Users may want to use their own header without the default header (disabled by `--no-header`), so they may use `--extra-header FOO --no-header`. (Current implementation)
Or we can let `--extra-header` automatically disable the default header (no timestamp, etc.), and this flag conflicts with `--no-header`.

Reply:
Change the option to just --header=, Since Extra indicates it is in addition to the existing header.

Reply:
I called it extra because the default header is also generated.
So, we would like to let the `--header` disable the default header and conflicts with --no-header, right?

Reply:
Yes

Reply:
I am against conflicting the two flags.  Look at the original issue https://github.com/containers/podman/issues/14072#issue-1222849370.  `--no-header` removes the standard header and `--extra-header`/`--header` appends.  This way, users can _add_ to custom header, create an entirely new one, or remove everything.